etc/env/ext-node.toml

```diff
@@ -48,6 +47,10 @@ mode = "FileBacked"
 file_backed_base_path = "artifacts"
 # ^ Intentionally set to coincide with main node's in order to read locally produced snapshots
 
+[en.main_node]
+url = "http://127.0.0.1:3050"
+rate_limit = { count = 10, window_ms = 30 }
```

Comment:
Why change defaults here?

Reply:
In general I'd say this rate limiting is more for us than for external partners

Reply:
- Changing defaults is not strictly necessary; just did it to showcase the new params and test that they are parsed by the EN.
- Not sure I get your point about the intended audience. If *any* EN is switched off for some time for whatever reason, it would likely hit the rate limit in the block fetcher; so the issue doesn't look specific to our ENs.